designsystem/src/commonMain/kotlin/org/hisp/dhis/mobileui/designsystem/theme/TextColor.kt

```diff
@@ -6,7 +6,7 @@ object TextColor {
     val OnSurface = Color.Charcoal800
     val OnSurfaceVariant = Color.Charcoal500
     val OnSurfaceLight = Color.Charcoal300
-    val OnError = Color.NeutralWhite
+    val OnError = Color.Red600
```

Comment:
Are you sure, In the specs it shows NeutralWhite
![image (5)](https://github.com/dhis2/dhis2-mobile-ui/assets/3330344/bae710e8-ad7d-4e99-900a-74d2947149b1)